src/matlab2tikz.m

```diff
@@ -4016,7 +4039,11 @@ function warnFacetedInterp(m2t)
 % ==============================================================================
 function [m2t, str] = drawStemSeries(m2t, h)
     [m2t, str] = drawStemOrStairSeries_(m2t, h, 'ycomb');
-
+    
+    % TODO: handle baseplane with stem3()
+    if m2t.axesContainers{end}.is3D
+      return
```

Comment:
No, ```m2t``` and ```str``` from ```drawStemOrStairSeries``` are still returned. In the 3D case, the next lines dealing with the baseline are skipped.